scivision/catalog/data/projects.json

```diff
@@ -0,0 +1,53 @@
+{
+  "catalog_type": "scivision project catalog",
+  "name": "default",
+  "entries": [
+    {
+      "name": "Tutorials",
+      "header": "Scivision tutorials project page",
+      "tasks": [
+        "classification",
+        "object-detection",
+        "segmentation"
+      ],
+      "models": [
+        "image-classifiers",
+        "huggingface-classifiers",
+        "huggingface-object-detection",
+        "huggingface-segmentation",
+        "butterfly-classification-model",
+        "flower-classification-model"
+      ],
+      "datasources": [
+        "data-003",
+        "data-005",
+        "flowers",
+        "butterflies"
+      ],
+      "notebooks": [
+        "https://github.com/scivision-gallery/scivision-basic-usage-examples/blob/main/how-to-use-scivision.ipynb",
+        "https://github.com/scivision-gallery/scivision-basic-usage-examples/blob/main/object-detection.ipynb",
+        "https://github.com/scivision-gallery/scivision-basic-usage-examples/blob/main/segmentation.ipynb",
+        "https://github.com/scivision-gallery/connections-workshop-examples/blob/main/ButterflyClassification.ipynb",
+        "https://github.com/scivision-gallery/connections-workshop-examples/blob/main/FlowerClassification.ipynb"
+      ],
+      "institution": [
+        "alan-turing-institute"
+      ],
+      "authors": [
+        "Ed Chalstrey"
```

Comment:
Unlike the models and datasources, it seems like an author (and contributors) is needed for projects - @ots22 what do you think? Alternatively we could remove this and rely on an out-link to have such information?

Reply:
Sounds like a good idea - I'll probably remove authors from the JSON for now and open a separate issue to look into the 'all contributors' model